util/sync_files.ts

```diff
@@ -43,14 +43,23 @@ const zoneReplace: ZoneReplace[] = [
         'ui/oopsyraidsy/data/06-ew/dungeon/another_sildihn_subterrane-savage.ts',
       'ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt':
         'ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane-savage.txt',
+
+      'ui/raidboss/data/03-hw/trial/sophia-ex.ts': 'ui/raidboss/data/06-ew/trial/sophia-un.ts',
```

Comment:
Would you mind adding an extra entry to the `zoneReplace` array rather than adding it into the criterion block directly?